README.md

```diff
@@ -102,7 +102,7 @@ It can also print the JSON to stdout:
 $ puppet strings some_manifest.pp --emit-json-stdout
 ```
 
-The schema for the JSON which Strings emits is [well documented](json_dom.md).
+The schema for the JSON which Strings emits is [well documented](https://github.com/puppetlabs/puppetlabs-strings/blob/master/json_dom.md).
```

Comment:
It does work on GitHub, but I wasn't sure about the forge. Better safe than sorry!